README.md

```diff
@@ -925,9 +925,9 @@ The identity visitor:
 Adding `:title` property to schemas:
 
 ```clj
-(m/accept
+(m/walk
   Address
-  (m/schema-visitor #(mu/update-properties % assoc :title (name (m/type %)))))
+  (m/schema-walker #(mu/update-properties % assoc :title (name (m/type %)))))
```

Comment:
Could this be wrapped by `m/walk` - ie. If we are always going to have to wrap the function with `m/schema-walker`, why not just have `m/walk` do it for us?

Similar to how the `m/encode` will auto coerce non-reified schemas using `m/schema`

Reply:
I think it covers the different cases nicely right now:

* `m/-walk` for any kind of walking, for library extensions
* `m/walk` with 4-arity fn for advanced postwalking (e.g. to map-syntax)
* `m/walk` + `m/schema-walker` to postwalk Schema instances to Schema instances, easy for many cases